src/main/interface/msp.c

```diff
@@ -280,9 +280,8 @@ static void serializeSDCardSummaryReply(sbuf_t *dst)
 static void serializeDataflashSummaryReply(sbuf_t *dst)
 {
 #ifdef USE_FLASHFS
-    if (flashfsIsSupported()) {
-        uint8_t flags = MSP_FLASHFS_FLAG_SUPPORTED;
-        flags |= (flashfsIsReady() ? MSP_FLASHFS_FLAG_READY : 0);
+    if (flashfsGetSize() > 0) {
+        uint8_t flags = (flashfsIsReady() ? 1 : 0) | 2 /* FlashFS is supported */;
```

Comment:
Don't we have define or `enum` for these constants?

Reply:
That was introduced in 3.4. I opted for making the minimal change that fixes the actual bug in 3.3, instead of pulling in more change from 3.4.